content/W2017/W17-tshirt-contest.md

```diff
@@ -0,0 +1,33 @@
+Title: T-Shirt Contest
+Date: 2017-02-26
+Category: Events
+Tags: social, big csters, panel
+Slug: tshirt-contest-w17
+Author: Monica Xu
+Summary: Design a T-Shirt for WiCS 
+
+
+Have a passion for design? 
+Love fashion? 
```

Comment:
if you want line breaks, you need an empty line between the lines. if you don't, you can just put this all on the same line